src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -994,7 +997,7 @@ function GlobeControls(view, target, domElement, viewerDiv, radius, getPickingPo
         if (this.enableDamping) {
             if (state === CONTROL_STATE.ORBIT && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
                 player.play(dampingOrbitalMvt).then(() => this.resetControls());
-            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && !(lastRotation[1].equals(lastRotation[0]))) {
+            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && (Date.now() - lastTimeMouseMove < 50) && !(lastRotation[1].equals(lastRotation[0]))) {
```

Comment:
Didn't you forget a word at the beginning of your comment (`disable` I suppose)?

And yes `GlobeControls` would benefit from being completely rewritten, taking advantages of itowns `Coordinates`.